sonar-project.properties

```diff
@@ -0,0 +1,4 @@
+sonar.projectKey=zlux:zlux-app-manager
+sonar.projectName=ZLUX Application Manager
+sonar.projectVersion=1.0.0
+sonar.sources=virtual-desktop/nodeServer,virtual-desktop/src,bootstrap/src
```

Comment:
Also, /system-apps/*/webClient, /system-apps/*/nodeServer (potentially) are needed, due to https://github.com/zowe/zlux-app-manager/tree/master/system-apps/app-prop-viewer

Reply:
I added properties for the system-apps I could pull down from master and changed the project version to 0.9.4 for now - longer term pulling the version from the source would be preferred. This is intended to be MVP - something that runs and you can check the output for.